bolt-sidecar/src/state/execution.rs

```diff
@@ -222,6 +230,7 @@ impl<C: StateFetcher> ExecutionState<C> {
             kzg_settings: EnvKzgSettings::default(),
             // TODO: add a way to configure these values from CLI
             validation_params: ValidationParams::new(gas_limit),
+            pricing: PreconfPricing::new(gas_limit),
```

Comment:
Nit but can we rename this to `pricer: InclusionPricer`? Less ambiguous if we have different preconf types in the future.